datacube_alchemist/upload.py

```diff
@@ -39,28 +46,35 @@ def upload_if_needed(self):
         If the data needs to be moved from a tmp location to s3 do the move.
         """
         if self.upload is True:
-            self.upload_now()
+            self.upload_now_change_control()
 
-    def upload_now(self):
-        fs = fsspec.filesystem('s3')
-        # fs.put(self._location, self.s3location, recursive=True)
+    def upload_now_change_control(self):
+        s3_resource = boto3.resource('s3')
```

Comment:
there is this in datacube-core now:

https://github.com/opendatacube/datacube-core/blob/develop/datacube/utils/aws/__init__.py#L327

includes handling of s3 urls, bu might not include ACLs, still would be nice to not duplicate code too much

Reply:
I talked to Kirill.  Let's get this fix in now and refactor later.